cli/medperf/entities/cube.py

```diff
@@ -73,24 +73,19 @@ def __init__(self, cube_dict):
         self.created_at = cube_dict["created_at"]
         self.modified_at = cube_dict["modified_at"]
 
-        cubes_storage = storage_path(config.cubes_storage)
-        self.cube_path = os.path.join(
-            cubes_storage, str(self.uid), config.cube_filename
-        )
-        self.params_path = None
-        if self.git_parameters_url:
-            self.params_path = os.path.join(
-                cubes_storage, str(self.uid), config.params_filename
-            )
-
         self.generated_uid = self.name
         path = storage_path(config.cubes_storage)
         if self.uid:
             path = os.path.join(path, str(self.uid))
         else:
             path = os.path.join(path, str(self.generated_uid))
-
+        # NOTE: maybe have these as @property, to have the same entity reusable
+        #       before and after submission
```

Comment:
Here you mean that once submitted, the path no longer reflects what it should be? If that's the case I totally agree! We should make this field dynamic so it gets updated after submission.